src/NHSDPublicBrowseAcceptanceTests.TestData/Utils/Queries.cs

```diff
@@ -104,6 +104,12 @@ from Solution
         public const string DeleteCatalogueItem = "DELETE FROM CatalogueItem WHERE CatalogueItemId=@CatalogueItemId";
         public const string GetAllCatalogueItems = "SELECT * FROM [dbo].[CatalogueItem]";
 
+        public const string GetFrameworkSolutionCount = @"SELECT COUNT(*)
+                                                              FROM FrameworkSolutions AS fs
+                                                              INNER JOIN CatalogueItem AS ci ON ci.CatalogueItemId = fs.SolutionId
+                                                              WHERE FrameworkId IN(SELECT TOP(1) Id FROM Framework WHERE ShortName = @frameworkName)
+                                                              AND ci.PublishedStatusId IN (3, 4)";
+
```

Comment:
Tend to recommend including the schema name when referencing tables, so `dbo.FrameworkSolutions` and `dbo.CatalogueItem`.